### ISSUE TITLE
Translation into Russian

### ISSUE THREAD

**Issue by Saimon398** [goldbergyoni/javascript-testing-best-practices#218]
Hello.
I would like to participate in such a cool project and translate into Russian? 
Is it still relevant?

**Comment by TREER00T** [comment]
> Hello. I would like to participate in such a cool project and translate into Russian? Is it still relevant?

Yes, if this project is not translated for the Russian language, you can be the first to translate it, if there are any, you can improve your language-specific translated projects, or you can start the translation again.
